### PR TITLE
Publish camera_info topic for rgb_points

### LINKED ISSUE #55: Why no images when I input rostopic echo /dope/rgb_points ？

### LINKED ISSUE #70: Lost some topic after start the dope.launch
I followed the tutorial , but I encountered some problem about rostopic and rviz.
First, I started  `roscore`

Second, I try to start my own camera(Intel RealSense D435i) node , the following is my camera.launch : 
      `roslaunch realsense2_camera rs_camera.launch`
After I start this node, I type `rostopic list` to check, here is some topic listed : 
`/camera/color/camera_info
/camera/color/image_raw
/camera/color/image_raw/compressed
/camera/color/image_raw/compressed/parameter_descriptions
/camera/color/image_raw/compressed/parameter_updates
/camera/color/image_raw/compressedDepth
/camera/color/image_raw/compressedDepth/parameter_descriptions
/camera/color/image_raw/compressedDepth/parameter_updates
/camera/color/image_raw/theora
/camera/color/image_raw/theora/parameter_descriptions
/camera/color/image_raw/theora/parameter_updates
`

Third, I followed the 3rd step on github :Edit config info (if desired) in `~/catkin_ws/src/dope/config/config_pose.yaml`
So I edited it, changing the `topic_camera` and `topic_camera_info` from default into the topic published by my own camera node, here is my change in `config_pose.yaml`:
`topic_camera: "/camera/color/image_raw"`
`topic_camera_info: "/camera/color/camera_info"`
`topic_publishing: "dope"`
`input_is_rectified: True`
`downscale_height: 500`
`frame_id: "/camera_link"`

Forth, after edited the `config_pose.yaml`, I started the dope node by `following:roslaunch dope dope.launch [config:=/path/to/my_config.yaml]`

Now, I type `rostopic list` again, I saw some topic about dope,
`/dope/detected_objects`
`/dope/dimension_cracker`
`/dope/markers`
`/dope/pose_cracker`
`/dope/rgb_points`

But I didn't see the `/dope/webcam_rgb_raw` , I open the `Rviz`, Add `Image`, `Pose`, `MarkerArray`, and `camera` , but only `camera ` not working, the cameara status is : `No image received.` . Here is the screenshot of my rviz : 
Screenshot

Is this condition normal ?

### PR DESCRIPTION
**Pull request by mintar**
...and also properly set the time stamp of rgb_points.

This allows visualizing the rgb_points using the Camera display in RViz.

See #119.

This fixes #55 and fixes #70.

Do not merge yet, I haven't been able to test it yet.

### CODE REVIEW

**Comment by Tingcccc** [comment]
I don't quite understand the meaning of setting the timestamp properly. Could you explain more?

**Comment by mintar** [comment]
Most ROS messages have a field called `header.stamp`. The old code did not set the header field at all. For the "Camera" display to work in RViz, there needs to be a `Image` and `CameraInfo` message that have the same header. This pull request copies the `header` (including the `header.stamp` field) from the `CameraInfo` to the `Image` message:

https://github.com/NVlabs/Deep_Object_Pose/pull/121/files#diff-043238b56ea02ad2847616a498debad2R310

In short: If you use this pull request, you don't need to do anything else; the "Camera" display should work for the `/dope/rgb_points` topic. Could you try that?

**Comment by Tingcccc** [comment]
I tried this pull request and it worked.

**Comment by mintar** [comment]
I have just successfully tested this PR, and it is ready for merging too.
